test/extended/util/client.go

```diff
@@ -137,9 +138,8 @@ func NewCLIWithoutNamespace(project string) *CLI {
 		adminConfigPath:  KubeConfigPath(),
 		withoutNamespace: true,
 	}
-	g.AfterEach(cli.TeardownProject)
-	g.AfterEach(cli.kubeFramework.AfterEach)
 	g.BeforeEach(cli.kubeFramework.BeforeEach)
+	g.JustAfterEach(cli.kubeFramework.AfterEach)
```

Comment:
kubeFramework.AfterEach deletes namespaces, so it cannot be `JustAfterEach`, it can be only `AfterEach`.

`cli.TeardownProject` works fine even if when it doesn't have a namespace. It deletes configs that are generated by ChangeUser.